src/kagglehub/gcs_upload.py

```diff
@@ -135,88 +136,23 @@ def _upload_blob(file_path: str, model_type: str) -> str:
     return response["token"]
 
 
-def upload_files(folder: str, model_type: str, quiet: bool = False) -> List[str]:  # noqa: FBT002, FBT001
-    """upload files in a folder. Zips the files if there are more than 50.
+def upload_files(folder: str, model_type: str) -> List[str]:
+    """Zips the files and upload them in a folder.
```

Comment:
nit: Zip and Upload directory

Quick suggestion on writing technical documents: https://developers.google.com/tech-writing/one

https://screenshot.googleplex.com/BuT8wShSVCz3ZSJ